conf/runner.yaml

```diff
@@ -75,6 +75,10 @@ runs:
         - status.services.mozilla.com
         - restmail.net
 
+        # Normandy
+        - normandy.cdn.mozilla.net
+        - self-repair.mozilla.org
```

Comment:
You don't want 14 days paging for this one? If so, it needs its own paging block.